src/engine/serverbrowser.cpp

```diff
@@ -526,8 +470,32 @@ void writeservercfg()
     f->printf("// servers which are connected to or queried get added here automatically\n\n");
     loopv(servers)
     {
-        serverinfo *s = servers[i];
-        f->printf("addserver %s %d %d %s %s %s %s\n", s->name, s->port, s->priority, escapestring(s->sdesc[0] ? s->sdesc : s->name), escapestring(s->authhandle), escapestring(s->flags), escapestring(s->branch));
+        servers[i]->writecfg( *f );
```

Comment:
Minor naming issue (this should be `write_config` or `write_cfg`, like the remaining methods).